app/lib/controlled_work_mappings/cw1_new.yml

```diff
@@ -0,0 +1,441 @@
+    - name: pensions_client
+      type: text
+      source: client_pensions
+
+    # Student Finance / Client
+    - name: studentfinance_client
+      type: text
+      source: client_student_finance
+
+    # Family or Friends / Client
+    - name: friendsandfamily_client
+      type: text
+      source: client_friends_and_family
+
+    # Property or Lodger / Client
+    - name: property_client
+      type: text
+      source: client_property_lodger_income
+
+    # Other Income / Client
+    - name: otherincome_client
+      type: text
+      source: client_other_income
+
+    # Benefits & Tax Credits / Client
+    - name: benefits_client
+      type: text
+      source: client_benefits
+
+    # Other income / Client
+    - name: undefined_50
+      type: text
+      source: client_non_employment_income
+
+    # Total gross income / Client
+    - name: undefined_52
+      type: text
+      source: client_total_income
+
+    # Total gross income  Client and Partner
+    - name: undefined_54
+      type: text
+      section: partner_income
+      source: total_combined_income
+
+    # Mortgage instalment / Client
+    - name: undefined_55
+      type: text
+      source: client_mortgage
+
+    # Rent / Client
+    - name: undefined_57
+      type: text
+      source: client_rent
+
+    # Dependents' allowances / Partner
+    - name: undefined_59
+      type: text
+      source: partner_allowance
+
+    # Dependents' allowances / Aged 15 or under
+    - name: undefined_60
+      type: text
+      source: dependants_allowance_under_16
+
+    # Dependents' allowances / Aged 16 or over
+    - name: undefined_61
+      type: text
+      source: dependants_allowance_over_16
+
+    # Tax and national insurance / Client
+    - name: undefined_62
+      type: text
+      source: client_tax_and_national_insurance
+
+    # Standard allowance for employment expense / Client
+    - name: undefined_64
+      type: text
+      source: client_employment_deduction
+
+    # Maintenance allowance / Client
+    - name: undefined_66
+      type: text
+      source: client_maintenance_allowance
+
+    # Childcare costs
+    - name: undefined_68
+      type: text
+      source: combined_childcare_costs
+
+    # Payment of income contribution order / Client
+    - name: undefined_69
+      type: text
+      source: client_legal_aid_contribution
+
+    # Total allowances / Client
+    - name: undefined_71
+      type: text
+      source: client_total_allowances
+
+    # Total monthly disposable income / Client
+    - name: undefined_73
+      type: text
+      source: client_disposable_income
+
+    # Total monthly disposable income Client and partner
+    - name: undefined_75
+      type: text
+      source: combined_disposable_income
+
+
+- section: partner_income
+  fields:
+
+    # Gross monthly earnings / Partner
+    - name: undefined_49
+      type: text
+      source: partner_gross_income
+
+    # Other income / Partner
+    - name: undefined_51
```

Comment:
I think we've missed the partner income mappings.

`undefined_51` no longer exists, we instead need to add in:
- benefits_partner
- maintenance_partner
- pensions_partner
- studentfinance_partner
- friendsandfamily_partner
- property_partner
- otherincome_partner